ocis/pkg/runtime/service/service.go

```diff
@@ -195,15 +204,43 @@ func Start(o ...Option) error {
 	// trap will block on halt channel for interruptions.
 	go trap(s, halt)
 
+	// add services with delayed execution.
 	time.Sleep(1 * time.Second)
+	scheduleServiceTokens(s, s.Delayed)
+
+	return http.Serve(l, nil)
+}
+
+// scheduleServiceTokens adds service tokens to the service supervisor.
+func scheduleServiceTokens(s *Service, funcSet serviceFuncMap) {
+	for _, name := range runset {
+		if _, ok := funcSet[name]; !ok {
+			continue
+		}
 
-	// add services with delayed execution.
-	for name := range s.Delayed {
 		swap := deepcopy.Copy(s.cfg)
-		s.serviceToken[name] = append(s.serviceToken[name], s.Supervisor.Add(s.Delayed[name](swap.(*ociscfg.Config))))
+		s.serviceToken[name] = append(s.serviceToken[name], s.Supervisor.Add(funcSet[name](swap.(*ociscfg.Config))))
 	}
+}
 
-	return http.Serve(l, nil)
+// generateRunSet interprets the cfg.Runtime.Extensions config option to cherry-pick which services to start using
+// the runtime.
+func (s *Service) generateRunSet(cfg *config.Config) {
+	if cfg.Runtime.Extensions != "" {
+		e := strings.Split(strings.Replace(cfg.Runtime.Extensions, " ", "", -1), ",")
```

Comment:
You could just use `strings.ReplaceAll` here.

Reply:
right that is just an alias for this 🙈